addons/l10n_tr_nilvera/lib/nilvera_client.py

```diff
@@ -38,13 +38,18 @@ def request(self, method, endpoint, params=None, json=None, files=None, handle_r
         start = datetime.utcnow()
         url = self.base_url + endpoint
 
-        response = self.__session.request(
-            method, url,
-            timeout=self.timeout_limit,
-            params=params,
-            json=json,
-            files=files,
-        )
+        try:
+            response = self.__session.request(
+                method, url,
+                timeout=self.timeout_limit,
+                params=params,
+                json=json,
+                files=files,
+            )
+        except (requests.ConnectionError, requests.Timeout) as e:
```

Comment:
```suggestion
        except requests.exceptions.RequestException as e:
```

It will catch everything